tests/playwright/checkout.spec.js

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from '@playwright/test'
+import { ProductPage } from './pages/ProductPage'
+import { CheckoutPage } from './pages/CheckoutPage'
+import { AccountPage } from './pages/AccountPage'
+
+const checkoutTypes = ['default']
+
+checkoutTypes.forEach((type) => {
```

Comment:
We don't need the foreach here as the checkout theme doesn't support the onestep checkout + is that the only difference in the tests here?